crates/astria-sequencer-relayer/local.env.example

```diff
@@ -44,6 +44,12 @@ ASTRIA_SEQUENCER_RELAYER_BLOCK_TIME=1000
 # Set to true to only relay blocks proposed by the validator key in `ASTRIA_SEQUENCER_RELAYER_VALIDATOR_KEY_FILE`.
 ASTRIA_SEQUENCER_RELAYER_RELAY_ONLY_VALIDATOR_KEY_BLOCKS=false
 
+# A comma-separated list of base64-encoded rollup IDs.
```

Comment:
We should probably explicitly spell out that rollup IDs are 32 byte hashes, in case somebody is confused (maybe "often the sha256 hash of a rollup name").

Reply:
Done in [cb1238b](https://github.com/astriaorg/astria/pull/1001/commits/cb1238b701b520d1bdbd778f544777afb9242736).